hardening/ansible/uefi.fmf

```diff
@@ -1,5 +1,7 @@
-tag+:
-  - broken
+adjust+:
+  - when: distro < rhel-10
+    enabled: false
+    because: RHEL-10+ is UEFI by default
```

Comment:
That's not the reason for *not* running it on RHEL-8 and 9. The real reason is
```
because: >
    we don't have capacity to test this everywhere, but RHEL-10 is UEFI
    by default, so run it at least there
```
(same in other tests)